### PR TITLE
libdap related cleanup

### PR DESCRIPTION
**Pull request by tjhei**
follow-up to #3334

### CODE REVIEW

**Comment by tjhei** [comment]
FYI: @kneumiller

File: contrib/opendap/OPeNDAP.md [hunk 1]
```diff
@@ -41,7 +41,11 @@ On OSX, the easy way is to use homebrew:
 
 > brew install libdap
 
-Skip down to the section on installing DEAL.II
+The library is alos available in Spack, see
```
Comment (gassmoeller) [inline contrib/opendap/OPeNDAP.md]:
```suggestion
The library is also available in Spack, see
```

File: source/utilities.cc [hunk 2]
```diff
@@ -1065,7 +1063,7 @@ namespace aspect
               AssertThrow(false,
                           ExcMessage(std::string("Reading of file ") + filename + " failed. " +
                                      "Make sure you have the dependencies for reading a url " +
-                                     "(run cmake with -DLIBDAP_ON=ON)"));
+                                     "(run cmake with -DASPECT_WITHLIBDAP=ON)"));
```
Comment (gassmoeller) [inline source/utilities.cc]:
```suggestion
                                     "(run cmake with -DASPECT_WITH_LIBDAP=ON)"));
```

**Comment by tjhei** [comment]
thanks for taking the time to spot the typos! Fixed.

**Pull request merged**